### DISCUSSION TITLE
Lock card

### DISCUSSION THREAD

**Discussion by alarno** [piitaya/lovelace-mushroom#493] [Q&A]
This is a very nice community. Congrats!

I have installed a lock card in my HA, but it's always locked, I have to unlock everytime to be able to click open. Is there any way to keep it unlocked? entity: nuki opener.

type: custom:mushroom-lock-card
entity: lock.porteria
name: Porteria
fill_container: false
hide_state: false

is there any other option to setup in yaml?, like locked: false...

Thanks in advance!

**Comment by piitaya** [comment]
Hi 🙂 
I didn't have lock so I don't know the behavior. Is the `open` service available even if the lock is `locked`?

**Reply by alarno** [reply]
Ok, let me get this right. I have a nuki opener, and, with your card, it appears by default blocked. If you click the lock key, you can unlock, and then, if appears 2 icons: Lock again, and a door to open, that door is what it would be very helpfull to have it directly as an option, otherwise, you have to unlock, wait some seconds, and open. Is this possible? I even tried this, adding a "locked:never", but, after a while, it comes back to locked:

type: custom:mushroom-lock-card
entity: lock.entity
name: entity
fill_container: false
hide_state: false
locked: never

**Reply by Radieserlbrot** [reply]
Hi piitaya,

I'm running into the same issue as the original post, four years later 🙂

**Lock:** Nuki Pro 5 (Home Assistant integration via MQTT, entity `lock.nuki_pro_5`).

Like the Nuki lock in this thread, my device handles the "locked → unlock → pull latch" sequence internally: calling the `lock.open` service works regardless of whether the lock is currently reported as `locked` or `unlocked` — the lock firmware unlocks first and then opens in one motion. So from the entity's perspective, "open" is a valid action in any state.

The mushroom-lock-card, however, only shows the open/unlatch icon once the entity state is already `unlocked`. While it's `locked`, only the unlock button is shown, which forces a "tap unlock → wait → tap open" flow even though a single `lock.open` call would do it directly.

For comparison, Home Assistant's native `type: tile` card with the `lock-commands` feature already shows the open button independent of the current lock state (as long as the entity reports the `open` supported feature) — tapping it just calls `lock.open` directly. I switched from that to mushroom-lock-card for the nicer look, but had to switch back because of this.

**Feature request:** an option to always show the open/unlatch control regardless of the current `locked`/`unlocked` state, similar to how the native tile card's `lock-commands` feature behaves — e.g. something like:

```yaml
type: custom:mushroom-lock-card
entity: lock.nuki_pro_5
show_open_always: true
```

Thanks for your efforts!
